app/models/feature_flag.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class FeatureFlag
+  def self.yaml_to_hash(filename)
+    yaml = YAML.parse_file(Rails.root.join(filename))
+    hash = yaml.presence&.to_ruby || {} # Handle empty yaml file
+    hash.with_indifferent_access.freeze
+  end
+
+  DEFINITION = yaml_to_hash("app/config/feature_flags.yaml") # rubocop:disable Layout/ClassStructure
```

Comment:
I guess it's okay to move the constant in the first list of this class to avoid the rubocop comment. 🙂

Reply:
I see :facepalm:... forget my comment :smile: